src/planet-a/PlanetGeometryA.cpp

```diff
@@ -38,21 +35,35 @@
 using namespace planeta;
 
 using osp::Vector3;
+using Magnum::Math::pow;
+using Magnum::Math::abs;
+
+void planeta::update_range_insert(std::vector<UpdateRangeSub>& range,
+                                  UpdateRangeSub insert)
+{
+    // TODO: Merge update ranges to decrease number of glBufferSubData calls
+
+    //UpdateRangeSub merged;
+
+    range.push_back(insert);
```

Comment:
UpdateRangeSub is a small object? No need for std::move() ?

Reply:
It's a struct of two uint32_t